models/location.js

```diff
@@ -9,6 +9,7 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Location.belongsToMany(models.Program, { through: models.ProgramLocation });
```

Comment:
This is an example of how the Many-to-Many relationship is set up in Sequelize in the models.